### DISCUSSION TITLE
TEMP CHECK: DEVCON & ETHGLOBAL BANGKOK 🇹🇭

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#194] [Proposals Temperature Check]
**Author:** Furqan Ahmed  
**Date:** 5th November 2024  

**Context:**  
I propose representing Darwinia Network at two essential blockchain events in Thailand: **Devcon 7** (November 11-15, 2024) and **ETHGlobal Bangkok** (November 15-17, 2024). Darwinia's strategic goal of onboarding more developers [REF](https://www.tally.xyz/gov/dcdao/proposal/29816049634754307523658710595374865934730721906103021309486532985405898633548) makes attending these conferences a high-priority initiative, given their reputation for attracting top-tier blockchain talent and fostering meaningful partnerships. By actively engaging at these events, I can showcase Darwinia’s potential to a diverse audience, build key partnerships, and position our network for sustainable growth.

To understand the impact and value I bring to such events, you can review my detailed report on Cosmoverse Dubai [here](https://github.com/orgs/dcdao/discussions/191), which highlights the contributions and connections made on behalf of Darwinia at the conference. Additionally, I have a strong network within the industry, ensuring I can secure access to both Devcon and ETHGlobal, even at this late stage.

**List of Conferences**
1. Devcon 2024
2. Pragma Bangkok
3. ETHGlobal Bangkok

**Budget Request:**  
To ensure impactful participation, I request a total budget of **$2,000 USD**, covering the following expenses:

**Flights:**  
- **Round-Trip (Karachi to Bangkok):** Approximately **$750 USD**.

**Accommodation (November 10-20th):**  
- **Hotel Costs:** Estimated at **$50-60 USD per night** for 11 nights, totaling approximately **$550-660 USD**. Giving a day before and a few extra days after the conferences to do post conference meetups and leaving time for any potential meeting i might have.

**Meals and Local Transportation:**  
- **Daily Expenses:** Allocating **$35 USD per day** for meals and local transportation over 11 days, totaling **$385 USD**.

**Misc**  
- **Miscellaneous Expenses:** Allocating $200 for various unexpected costs.

**Total Request APPROX: $2,000 USD**

**Justification:**  
Attending both Devcon 7 and ETHGlobal Bangkok is crucial for Darwinia’s developer engagement and partnership goals [REF](https://www.tally.xyz/gov/dcdao/proposal/29816049634754307523658710595374865934730721906103021309486532985405898633548) . These events provide invaluable opportunities to showcase Darwinia’s offerings, onboard talented developers, and foster connections within the blockchain community. The proposed budget will ensure active and meaningful participation, maximizing Darwinia’s impact and visibility in the region.

**Comment by boundless-forest** [comment]
Nice to see that our brand can be showcased in those top-tier meetings. I have a suggestion: we should focus on the RingDAO market instead of just the Darwinia network. It would be better to represent RingDAO at those events, as we have more to connect with a broader community.

**Reply by nblogist** [reply]
When I say Darwinia - it encompasses any related products - the problem is that most people don't know ring that's why they default to talking about Darwinia not RIngDAO - but I am happy to work on changing that narrative - can you share more what your idea is looking like

**Comment by hackfisher** [comment]
ETHGlobal Bangkok is scheduled for November 15–17, 2024, according to their website: https://ethglobal.com/events/bangkok, not November 22–24, 2024.

Considering the tight timeline, the event will already be underway when the proposal is approved. Itering and Helixbox will also be sending team members to this event during November 8 and November 17, so if there are relevant meetings or activities, please feel free to request support from Itering’s development team, they can help to save cost in this case. For future events, it would be beneficial to plan and arrange things earlier.

I will vote against it unless there is an explanation of what the itinerary from the 17th to the 25th is for.

**Reply by nblogist** [reply]
Thank you so much for pointing this out! I mixed up the dates and calculations. They are now fixed and ready for review. And yes in future we will plan before hand - its just that as we are just starting out right now plus things are popping up pretty close too. Once this proposal passes - please make a group and I can even help get the teams into various side events my friends will host.

**Reply by nblogist** [reply]
Moving the corrected dates version to Tally!

**Comment by nblogist** [comment]
Published on Tally! [Here](https://www.tally.xyz/gov/dcdao/proposal/48582902085907621979427150849355865514372129950147595919220816900759113235962)
